src/status_im/contexts/wallet/events.cljs

```diff
@@ -202,6 +202,32 @@
                                     (first derived-address-details)]))]
      {:fx [[:dispatch [:wallet/create-derived-addresses account-details on-success]]]})))
 
+(defn finalize-new-keypair
+  [_ [{:keys [sha3-pwd new-keypair]}]]
+  {:fx [[:json-rpc/call
+         [{:method     "accounts_addKeypair"
```

Comment:
it would be a nice practice if we can start adding contract tests for all new rpc calls. It can be done in a follow up if preferred, but can an issue to track this be created if done that way?

Reply:
accounts_addKeypair - this method also creates the account?
perhaps the name `finalize-new-keypair` could be more informative - add-new-keypair-and-create-account (same for it's corresponding event name

Reply:
@J-Son89 open an issue to add contract tests: https://github.com/status-im/status-mobile/issues/19019

Reply:
Thanks @OmarBasem! 🙏